projects/js-packages/charts/src/components/line-chart/stories/config.tsx

```diff
@@ -1,32 +1,76 @@
+import { GlyphDiamond, GlyphStar } from '@visx/glyph';
 import React from 'react';
+import { jetpackTheme, wooTheme, ThemeProvider } from '../../../providers/theme';
+import { DefaultGlyph } from '../../shared/default-glyph';
 import LineChart from '../line-chart';
 import sampleData from './sample-data';
 import type { Meta } from '@storybook/react';
 
+const customStorybookTheme = {
+	...jetpackTheme,
+	glyphs: [
+		props => React.createElement( DefaultGlyph, { ...props, key: props.key } ),
+		props =>
+			React.createElement( GlyphStar, {
+				key: props.key,
+				top: props.y,
+				left: props.x,
+				size: props.size * props.size,
+				fill: props.color,
+			} ),
+		props =>
+			React.createElement( GlyphDiamond, {
+				key: props.key,
+				top: props.y,
+				left: props.x,
+				size: props.size * props.size,
+				fill: props.color,
+			} ),
+	],
+};
+
+const THEME_MAP = {
+	default: undefined,
+	jetpack: jetpackTheme,
+	woo: wooTheme,
+	customStorybook: customStorybookTheme,
+};
+
 export const lineChartMetaArgs = {
 	title: 'JS Packages/Charts/Types/Line Chart',
 	component: LineChart,
 	parameters: {
 		layout: 'centered',
 	},
 	decorators: [
-		Story => (
-			<div
-				style={ {
-					resize: 'both',
-					overflow: 'auto',
-					padding: '2rem',
-					width: '800px',
-					maxWidth: '1200px',
-					border: '1px dashed #ccc',
-					display: 'inline-block',
-				} }
-			>
-				<Story />
-			</div>
-		),
+		( Story, { args } ) => {
```

Comment:
This new config was added to index.stories.tsx in https://github.com/Automattic/jetpack/pull/43875, moving it to the shared config.